Vic2ToHoI4/Data_Files/configurables/converterFocuses.txt

```diff
@@ -3118,24 +3119,27 @@ focus = {
 		}
 		ai_will_do = {
 			factor = 1
+			modifier = {
+				factor = 0
+				$TARGET = { in_weak_faction = no }
+			}
 			modifier = {
 				factor = 0
 				date < 1938.8.1
 			}
 			modifier = {
 				factor = 0
 				threat > 0.25
```

Comment:
To confirm, we want to block the ai from taking this if threat is too high too early?
Or do we want it the other way around? The ai requires a sufficient threat to be willing to take this?

Reply:
We want the AI to take it if the threat is too low, thus increasing it slightly. I think I got the logic right this time, but feel free to correct me.

Reply:
The logic should be correct then. I just wasn't sure if that was the intent.